spot-client/src/common/app-state/remote-control-service/actions.js

```diff
@@ -42,11 +50,15 @@ function createActionWithRequestStates( // eslint-disable-line max-params
     ));
 
     return request()
-        .then(() => dispatch(setRequestState(
-            requestType,
-            requestStates.DONE,
-            expectedValue
-        )), error => {
+        .then((...args) => {
```

Comment:
hmm what happened here ? maybe the docs need to be updated to explain this ?

Reply:
yeah it would be good to mention that `@returns {Function<Promise>}` is resolved with the result of `request`